odoo/models.py

```diff
@@ -6853,6 +6860,7 @@ def _apply_onchange_methods(self, field_name, result):
                     res['warning'].get('type') or "",
                 ))
 
+    #@api.readonly?
```

Comment:
If you don't know, simply don't add this line.
We might enforce it later, but we know that it currently doesn't work in some cases.